cli/commands/run/index.ts

```diff
@@ -11,16 +12,26 @@ import { RunProdServer } from './server';
 
 export default function provideRun(
   container: AwilixContainer,
+  ethersProvider: providers.JsonRpcProvider,
+  chainIds: number[],
   cache: Cache,
   args: any
 ): CommandHandler {
   assertExists(container, 'container')
   assertExists(cache, 'cache')
+  assertExists(chainIds, 'chainIds')
+  assertExists(ethersProvider, "ethersProvider");
   assertExists(args, 'args')
 
   return async function run(runtimeArgs: any = {}) {
     args = { ...args, ...runtimeArgs }
 
+    const network = await ethersProvider.getNetwork();
+
+    if(!network) throw Error(`No network detected at rpc url.`);
```

Comment:
i would specifically mention the term jsonRpcUrl since that is something devs will recognize i.e. `No network detected at jsonRpcUrl`